crates/shared/src/price_estimation/trade_finder.rs

```diff
@@ -344,7 +344,7 @@ impl Clone for TradeEstimator {
     fn clone(&self) -> Self {
         Self {
             inner: self.inner.clone(),
-            sharing: Default::default(),
```

Comment:
I was not aware of this particular detail of the `Clone` implementation.
I believe this means that currently `verified` request will not be shared with `fast` or `optimal` requests.
That makes sense (because `verified` might return something else than `optimal`) but once we move to verified quotes completely we might see little to no use for `RequestSharing` because of it.